### DISCUSSION TITLE
PAGE-ing all Community on Discussions members 📄 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#30830] [New to GitHub]
Welcome back for another addition of the New to GitHub Check-in 🗞️ We’ve got community members to spotlight, Community content you don’t want to miss, and the latest from the GitHub blog. 

Let’s dig in! 

## 🌟Community Superstars: A round of applause for these community citizens 👏
* @hibixby: Did some great sleuthing 🕵️‍♀️ to share why a [configuration](https://github.com/orgs/community/discussions/29771) wasn’t working - thank you! 
* @melvincwng: Helpful and speedy, 14 minutes is all it took for them to [answer](https://github.com/orgs/community/discussions/30668#discussion-4330262)  complete with screenshot! 
* @halldorr: For asking a great question, GitHub’s products are not one size fits all and everyone needs something different. Appreciate you starting this [conversation](https://github.com/orgs/community/discussions/29964). 

## 💬 All Around Discussions… New and noteworthy!
* 🗒️ Introducing the Pages Category: Pages now has its own [category](https://github.com/orgs/community/discussions/categories/pages?discussions_q=sort%3Atop+created%3A%3E%3D2022-08-17+category%3APages) in Community on Discussions, use it to ask questions, share feedback, and collaborate with other GitHub Pages users.
* 👩‍✈️ Copilot AMA: Can you believe it has been just two months since Copilot launched? We answered top of mind questions in our AMA that ended earlier. Missed the action? [See what was asked](https://github.com/orgs/community/discussions/29932). 
* 🌐 Meet up with fellow developers and tech enthusiasts: @andreagriffiths11 [posted](https://github.com/orgs/community/discussions/29844) about how to connect! 

## 📖 GitHub Reads: Check out these posts to keep up with GitHub! 
* 🌞 Release Radar: July 2022: [Curated list](https://github.blog/2022-08-05-release-radar-jul-2022/) of projects that released major version updates. Consider using them or even contributing to them! 
* ✔️ Customize Your Default Commit Message: New options have been added when merging pull requests, [review the changelog](https://github.blog/changelog/2022-08-23-new-options-for-controlling-the-default-commit-message-when-merging-a-pull-request/) to see how. 
* 🍴Improved UI for Syncing a Fork: [Learn how](https://github.blog/changelog/2022-08-24-improved-ui-for-syncing-a-fork/) the UI has been updated! 

That is all from now, but let us know if anything is blocking you on your GitHub journey in the comments below and we’ll try our best to help!

**Comment by ettaboyle** [comment]
Thanks & cheers @HiBixby @melvincwng & @halldorr ✨ 👏🏻

**Reply by flash55553** [reply]
can someone help me!

**Reply by flash55553** [reply]
my account was hacked and I really want it to get back can someone pls help me!

**Reply by queenofcorgis** [reply]
Hi there @flash55553 ! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Knoxxy9050** [reply]
Ok, i am help you

**Reply by Rizhack777** [reply]
Tempat apa ini co

**Comment by maxamedmukhtaar100** [comment]
thanks alot off all my friend! Am from somalia, my name mohamed moktari.
just i want learn in programing language but i don't to execute some
programs like C, C++, Python and so on.
`please if exist person to help me that's very good Idea!

**Reply by queenofcorgis** [reply]
Welcome @maxamedmukhtaar100 👋🏼 Do you have a programming language in mind or are you still deciding?

**Comment by Gatito1010** [comment]
Good evening, I just opened another account, my previous account was gato1010e, they were looking for a repository and with the person I knew, when I gave him my name and surname, he told me that he had money

**Comment by Gatito1010** [comment]
Apellido me dijeron que se comunicarían con migo pero ya no lo me ablaron solo me mintieron o hay algo de verdad en eso por que me pidieron número de cuenta bancaria pero no la mande

**Comment by Gatito1010** [comment]
Mi nombre es Israel Hernández soy de Morelos México mi cuenta actual es gatito1010 la otra ya no la pude recuperar que era gato1010

**Reply by queenofcorgis** [reply]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
